sky/backends/cloud_vm_ray_backend.py

```diff
@@ -3508,12 +3508,13 @@ def _execute(
         task_copy = copy.copy(task)
         # Handle multiple resources exec case.
         task_copy.set_resources(valid_resource)
+        logger.info(valid_resource)
```

Comment:
Should we elaborate this output? It would be a bit surprising to see just a resources in the output.
For example:
```suggestion
        if len(task.resources) > 1:
            logger.info(f'Multiple resources are specified for the task, using: {valid_resource}')
```